src/axom/spin/internal/linear_bvh/vec.hpp

```diff
@@ -36,8 +36,10 @@ class Vec
   T m_data[S];
 
   //
-  //  No contructors so this is a POD type
+  //  Use only default contructors to keep this a POD type
```

Comment:
@gzagaris -- I verified that `Vec` is still a POD type with `default` constructors using 
```cpp
   static_assert(std::is_pod< Vec >::value, "Vec<T,S> should be a POD");
```
but didn't keep this static_assert in the code.

See: https://stackoverflow.com/questions/7169658/is-this-struct-pod-in-c11